cmd/ingest/main.go

```diff
@@ -22,20 +27,30 @@ var logger = logrus.New()
 var db *sql.DB
 
 type Options struct {
-	zmqAddr  string
-	zmqTopic string
 	dbPath   string
 	logLevel uint64
 	logPath  string
+	zcashConfPath string
+	startHeight uint64
+	rpcHost string
+	rpcPort string
+	rpcLogin string
+	rpcPass string
 }
 
 func main() {
 	opts := &Options{}
-	flag.StringVar(&opts.zmqAddr, "zmq-addr", "127.0.0.1:28332", "the address of the 0MQ publisher")
-	flag.StringVar(&opts.zmqTopic, "zmq-topic", "checkedblock", "the stream to listen to")
 	flag.StringVar(&opts.dbPath, "db-path", "", "the path to a sqlite database file")
 	flag.Uint64Var(&opts.logLevel, "log-level", uint64(logrus.InfoLevel), "log level (logrus 1-7)")
 	flag.StringVar(&opts.logPath, "log-file", "", "log file to write to")
+	flag.StringVar(&opts.zcashConfPath, "conf-file", "", "conf file to pull RPC creds from")
+	flag.Uint64Var(&opts.startHeight, "start-height", uint64(0), "start sync from this height (> 419200 for mainnet)")
+	flag.StringVar(&opts.rpcHost, "rpc-host", "", "RPC host of zcashd")
+	flag.StringVar(&opts.rpcPort, "rpc-port", "", "RPC port of zcashd")
+	flag.StringVar(&opts.rpcLogin, "rpc-login", "", "RPC login of zcashd")
+	flag.StringVar(&opts.rpcPass, "rpc-pass", "", "RPC password of zcashd")
```

Comment:
this seems to be accepting the RPC password as a command-line option. If that's the case, then this password could be read from the process status info. It's probably better to leave these values in the `zcash.conf` file, since we need to access that file, anyway, and access to that file can be restricted via file permissions.